### PR TITLE
First backend steps

### PR DESCRIPTION
**Pull request by alexbagirov**
As of now, we didn't have any app logic. This commit adds the database creation code and the first Intro page interaction. After filling in the class and hitting the Start button the app will send new user's info to OneSignal and populate the database.

**Important**: The invisible `cordova.js` file was added. This one should fix all Onsen problems like white screens and others.

### CODE REVIEW

**Comment by alexbagirov** [comment]
Ready for review.

File: www/js/common.js [hunk 1]
```diff
@@ -12,10 +12,25 @@ window.fn.load = function(page) {
...
+
+// Set up the schema
+db.transaction(function (tx) {
+  tx.executeSql('CREATE TABLE IF NOT EXISTS settings (key TEXT, value TEXT)', []);
+});
+
+// Init OneSignal
+document.addEventListener('deviceready', function () {
+  // What happens when notification is received
+  var notificationOpenedCallback = function(jsonData) {
```
Comment (Leva7) [inline www/js/common.js]: Why not do it like `function notificationOpenedCallback(jsonData)`? That seems to be clearer

File: www/js/index.js [hunk 2]
```diff
@@ -116,8 +116,6 @@ function jumpToCarouselPage() {
 }
 
 // Set the default page to open
```
Comment (Leva7) [inline www/js/index.js]: Remove the comment, too

File: www/js/intro.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+function checkIntroFields() {
+  var carousel = document.querySelector('ons-carousel');
+
+  var introName = document.querySelector('#intro-name').value;
+  var introClass = document.querySelector('#intro-class').textContent;
```
Comment (Leva7) [inline www/js/intro.js]: There's a method named `getElementById`. Not sure if it's more efficient, but at least it's more explicit, so I'd prefer that over the selector

File: www/js/intro.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+  if (introClass == 'Выберите класс') {
```
Comment (Leva7) [inline www/js/intro.js]: Don't know the whole deal with equality operators in JS, but I'm pretty sure `==` is roughly equal to `is` in Python. Should we perhaps use `===`, which is the actual equality operator?

File: www/js/common.js (hunk 1 above)
Comment (alexbagirov) [inline www/js/common.js]: This is an example from official OneSignal repo. Nothing special here.

File: www/js/intro.js (hunk 4 above)
Comment (alexbagirov) [inline www/js/intro.js]: Google says that it is fine.

**Pull request merged**